inference-engine/tests/functional/inference_engine/pdpd_reader/read_pdpd_model_test.cpp

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <set>
+#include <string>
+#include <fstream>
+
+#include <ie_blob.h>
+#include <ie_core.hpp>
+#include <file_utils.h>
+#include <ngraph/ngraph.hpp>
+
+TEST(PDPD_Reader_Tests, ImportBasicModelToCore) {
+    auto model = std::string(PDPD_TEST_MODELS) + "relu.pdmodel";
+    InferenceEngine::Core ie;
+    auto cnnNetwork = ie.ReadNetwork(model);
+    auto function = cnnNetwork.getFunction();
+
+    int count_relus = 0;
+    int count_constants = 0;
+    int count_parameters = 0;
+
+    for (auto op : function->get_ops()) {
```

Comment:
I think we can create reference nGraph Function in the code and compare this function with function from the reader.
In this case the comparator can check operation, tensor names, runtime info and etc.